AssertionError: invalid document für example content

Aufrufen der document overview für das als Teil des Example Contents erstellte Dokument "Vorbericht Finanzplan" führt zu einem `AssertionError`.

Steps to reproduce:
- Von aktuellem `master` mit example content aufsetzen
- http://localhost:8080/fd/sitzungen/committee-3/submitted-proposal-75/document-19 aufrufen.

```pytb
  Module zope.tales.pythonexpr, line 59, in __call__
   - __traceback_info__: ( view.is_outdated(submitted_document))
  Module <string>, line 1, in <module>
  Module opengever.document.browser.overview, line 162, in is_outdated
  Module opengever.meeting.model.submitteddocument, line 52, in is_up_to_date
AssertionError: invalid document
```

--- a/docs/HISTORY.txt
+++ b/docs/HISTORY.txt
@@ -5,6 +5,9 @@ Changelog
 4.5.0 (unreleased)
 ------------------
 
+- Only display link to proposal for documents in repositories.
+  [deiferni]
+
 - Show statusmessage after successfully submitting a proposal.
   [phgross]
 

--- a/opengever/document/browser/overview.py
+++ b/opengever/document/browser/overview.py
@@ -147,7 +147,7 @@ class Overview(DisplayForm, OpengeverTab):
             yield data
 
     def display_submitted_documents(self):
-        return is_meeting_feature_enabled() and self.submitted_documents
+        return is_meeting_feature_enabled() and self.submitted_documents()
 
     def submitted_documents(self):
         return SubmittedDocument.query.by_document(self.context).all()

--- a/opengever/document/browser/overview.py
+++ b/opengever/document/browser/overview.py
@@ -150,7 +150,7 @@ class Overview(DisplayForm, OpengeverTab):
         return is_meeting_feature_enabled() and self.submitted_documents()
 
     def submitted_documents(self):
-        return SubmittedDocument.query.by_document(self.context).all()
+        return SubmittedDocument.query.by_source(self.context).all()
 
     def get_update_document_url(self, submitted_document):
         return '{}/@@submit_additional_document?submitted_document_id={}'.format(

--- a/opengever/document/tests/test_overview.py
+++ b/opengever/document/tests/test_overview.py
@@ -263,6 +263,21 @@ class TestOverviewMeetingFeatures(FunctionalTestCase):
                          proposal.css('a').first.get('href'))
 
     @browsing
+    def test_submitted_proposal_link_is_not_shown_for_submitted_documents(self, browser):
+        committee = create(Builder('committee'))
+        proposal = create(Builder('proposal')
+                          .within(self.dossier)
+                          .having(title='Mach doch',
+                                  committee=committee.load_model())
+                          .relate_to(self.document)
+                          .as_submitted())
+        submitted_document = proposal.load_model().submitted_documents[0].resolve_submitted()
+
+        browser.login().open(submitted_document, view='tabbedview_view-overview')
+        proposals = browser.css('#proposals_box .proposal')
+        self.assertEqual(0, len(proposals))
+
+    @browsing
     def test_outdated_document_can_be_updated(self, browser):
         # create a new document version
         repository = api.portal.get_tool('portal_repository')

--- a/opengever/meeting/model/query.py
+++ b/opengever/meeting/model/query.py
@@ -72,6 +72,9 @@ class MembershipQuery(BaseQuery):
 class SubmittedDocumentQuery(BaseQuery):
 
     def get_by_source(self, proposal, document):
+        """Get the one submitted document for proposal and document.
+        """
+
         oguid = Oguid.for_object(document)
         proposal_model = proposal.load_model()
         return self.filter(self._attribute('oguid') == oguid)\
@@ -81,6 +84,14 @@ class SubmittedDocumentQuery(BaseQuery):
     def get_by_target(self, document):
         oguid = Oguid.for_object(document)
         return self.filter(self._attribute('submitted_oguid') == oguid).first()
+
+    def by_source(self, document):
+        """Return all submitted documents where document is on the source
+        side.
+        """
+
+        oguid = Oguid.for_object(document)
+        return self.filter(self._attribute('oguid') == oguid)
 
     def by_document(self, document):
         """Filter by document's oguid on source or target side of a submitted

